evalml/tests/utils_tests/test_cli_utils.py

```diff
@@ -21,8 +21,13 @@ def current_dir():
     return os.path.dirname(os.path.abspath(__file__))
 
 
-def test_get_core_requirements():
-    assert len(get_core_requirements()) == 15
+def get_core_requirements(current_dir):
+    reqs_path = os.path.join(current_dir, '../../../core-requirements.txt')
```

Comment:
Two notes:
1) This path won't work on windows, right? Interesting that the windows test still passed... Either way, safest to use `pathlib`: `pathlib.Path('..', '..', '..', 'core-requirements.txt')`
2) I thought there was a way to have pytest automagically handle this for you somehow? Although I just spent 2min googling and couldn't find it 😅

Reply:
@dsherry I spent some time looking at this on the circle-ci windows vm and I have not explanation as to why this works...but it does lol.

Running os.path.join with these inputs returns a string with mixed windows and linux separators - so it should cause a problem:

![image](https://user-images.githubusercontent.com/41651716/99092925-d3e59b80-259f-11eb-8101-deea322204ee.png)

But then `open` just handles it like a champ - no idea how it works. Must be doing something to resolve the path under the hood. 

![image](https://user-images.githubusercontent.com/41651716/99093014-efe93d00-259f-11eb-861a-65138067a24e.png)

Needless to say, I'll just change this to pathlib lol.